tests/testthat/test-check_tbl_values_required.R

```diff
@@ -251,3 +251,33 @@ test_that("Ignoring derived_task_ids in check_tbl_values_required works", {
     )
   )
 })
+
+test_that("check_tbl_values_required works with all optional output types", {
```

Comment:
Adding the issue number for the test so that we can be alerted to the context immediately if something breaks again. 

```suggestion
test_that("(#123) check_tbl_values_required works with all optional output types", {
```